### ISSUE TITLE
Error when trying to start

### ISSUE THREAD

**Issue by FelipePipe2002** [rdeepak2002/reddit-place-script-2022#96]
# Bug Report Form

## Checklist

- [y] I am using the latest version of the code.
- [y] I am absolutely sure I read the README in its entirety.
- [y] I have made sure this is not a duplicate issue.
- [y] I have filled out this form correctly.

## Describe the bug
getting this error "An error occured. Make sure you have the correct credentials. Response data: {'message': 'Unauthorized', 'error': 401}"

## To Reproduce

Steps to trigger the bug:

1. Go to 'cmd'
2. type  'python main.py'
3. See error

## Expected behavior
start runing

**Comment by esthrshi** [comment]
Did you fill in all credentials (username, password, app key and secret) in correctly, and are you sure they're in the right format?
You are getting this error most likely because one of those things aren't correct.

**Comment by FelipePipe2002** [comment]
yes, I checked again and everything is fine, I don't know why it happens.

**Comment by esthrshi** [comment]
Did you rename your `config_example.json` to `config.json` ?

**Comment by FelipePipe2002** [comment]
yep

**Comment by FelipePipe2002** [comment]
![image](https://user-images.githubusercontent.com/64708520/161398338-ae0c51dc-690a-4cfc-ba7b-dcbeb0a4a742.png)
that are the things that i change
1° Coords
2° Name
3° Password
and so on

**Comment by esthrshi** [comment]
Your client ID seems unusually short, are you sure you are using the right one? It should be around 22 characters.

**Comment by esthrshi** [comment]
Your client ID can be found under 'personal use script'

**Comment by FelipePipe2002** [comment]
ahhhhhh, i have to change it, my mistake, thanks! now I have another error, I don't know why.

**Comment by esthrshi** [comment]
What is the error?

**Comment by FelipePipe2002** [comment]
![image](https://user-images.githubusercontent.com/64708520/161398580-ddbbf57d-5081-4843-9726-ca60b943e14b.png)

**Comment by esthrshi** [comment]
Your formatting in `config.json` is probably not correct, double check that everything is in double quotes ("").

It seems your password is in single quotes? If so you need to change it to `"password"`

**Comment by FelipePipe2002** [comment]
i fix it, now i get "Response data: {'error': 'invalid_grant'}"

**Comment by DWAA1660** [comment]
> ![image](https://user-images.githubusercontent.com/64708520/161398580-ddbbf57d-5081-4843-9726-ca60b943e14b.png)

i have this issue everything is in double quotes to my knowledge

**Comment by rdeepak2002** [comment]
Hello, "invalid_grant" means that there is an authentication issue. I also suggest using the latest code which only requires username and password.

**Comment by rdeepak2002** [comment]
Hello, were you able to get your config working now? I would also suggest using the latest code which only requires username and password.
